docs/rules/no-root-v-if.md

```diff
@@ -0,0 +1,39 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-root-v-if
+description: enforce valid `v-if` directives on root element
+---
+# vue/no-root-v-if
+
+> enforce valid `v-if` directives on root element
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+This rule checks whether every template root is valid.
+
+## :book: Rule Details
+
+This rule reports the template root in the following cases:
+
+<eslint-code-block :rules="{'vue/no-root-v-if': ['error']}">
+
+```vue
+<!--  `v-if` should not be used on root element without `v-else` -->
+<template>
+  <div v-if="foo"></div>
+</template>
+
+<template><custom-component v-if="shouldShow" /></template>
+```
+
+</eslint-code-block>
```

Comment:
Two `<template>` blocks break the example. I think one should suffice to show the point:

````suggestion
<eslint-code-block :rules="{'vue/no-root-v-if': ['error']}">

```vue
<template>
  <div v-if="foo"></div>
</template>
```

</eslint-code-block>
````